doc/POINTS_COORDINATES.md

```diff
@@ -0,0 +1,169 @@
+# Points, tripoints, and coordinate systems
+
+## Coordinate systems
+
```

Comment:
somewhere you should mention that the X axis normally goes from left to right of the screen (also west to east on a north oriented map), and that the Y axis normally goes from top to bottom of the screen (also north to south on a north oriented map).

Reply:
Added at the top.